client/layout/index.jsx

```diff
@@ -98,22 +96,27 @@ const Layout = createReactClass( {
 	},
 
 	render: function() {
+		const { currentRoute } = this.props;
 		const sectionClass = classnames(
-				'layout',
-				'color-scheme',
-				`is-${ this.props.colorSchemePreference }`,
-				`is-group-${ this.props.section.group }`,
-				`is-section-${ this.props.section.name }`,
-				`focus-${ this.props.currentLayoutFocus }`,
-				{ 'is-support-user': this.props.isSupportUser },
-				{ 'has-no-sidebar': ! this.props.hasSidebar },
-				{ 'has-chat': this.props.chatIsOpen },
-				{ 'has-no-masterbar': this.props.masterbarIsHidden }
-			),
-			loadingClass = classnames( {
-				layout__loader: true,
-				'is-active': this.props.isLoading,
-			} );
+			'layout',
+			'color-scheme',
+			`is-${ this.props.colorSchemePreference }`,
+			`is-group-${ this.props.section.group }`,
+			`is-section-${ this.props.section.name }`,
+			`focus-${ this.props.currentLayoutFocus }`,
+			{ 'is-support-user': this.props.isSupportUser },
+			{ 'has-no-sidebar': ! this.props.hasSidebar },
+			{ 'has-chat': this.props.chatIsOpen },
+			{ 'has-no-masterbar': this.props.masterbarIsHidden }
+		);
+		const loadingClass = classnames( {
+			layout__loader: true,
+			'is-active': this.props.isLoading,
+		} );
+		const isJetpackConnectSection = 'jetpack-connect' !== this.props.section.name;
```

Comment:
@tyxla, this is my take on the additions made in #26921, see [line 159](https://github.com/Automattic/wp-calypso/pull/26955/files#diff-e8ab61833a6d85e586cf195046ed6f15R159) also.

Does this still do as it should?